.github/workflows/haskell.yml

```diff
@@ -1,43 +1,77 @@
-name: Haskell CI
-
+name: build
 on:
   push:
-    branches: [ "main" ]
+    branches: [main, master]
```

Comment:
there isn't `master` branch. curious why add `master` here?

Reply:
Oh mostly juts copy pasted, missed this, thank you!